core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -301,12 +301,18 @@ oppia.factory('ExplorationPlayerService', [
         return exploration.getUninterpolatedContentHtml(stateName);
       },
       getStateContentAudioTranslations: function(stateName) {
+        if (stateName === null) {
```

Comment:
Actually, concept cards are supposed to have audio translations...

Again, it would be better if this and other functions in this file aren't called at all when we're in concept card mode. Instead we would call a ConceptCardService or similar to provide the necessary data.

Reply:
Done, removed this check and added a check where this function is called. When translations are added in the future, the corresponding function for concept cards can be added there.